README.md

```diff
@@ -21,8 +21,10 @@ Anyone can comment on a paper's PDF on [GDrive][GDrive] with questions or things
 clarifying.
 
 
-## Next session
-__2018.10.12__ at 4pm CET, paper TBD in [src-d/reading-club#7](https://github.com/src-d/reading-club/issues/7)
+## Next sessions
+-__Extra session on 2018.10.5__ at 4pm CET, we'll study [Code Vectors: Understanding Programs Through Embedded Abstracted Symbolic Traces](https://drive.google.com/open?id=1NyGQXxF2Ctq21wYcMjFWGyW0mwmDrXkD). __We will be joined by Jordan Henkel, first author of the paper!__ Please write any questions you might have in [this GDoc](https://docs.google.com/document/d/1TjiRzSivxXDDl79Fq4Bs7Ax3Zz7hciRGPKxKrUHFd6I/edit).
```

Comment:
How about changing wording/formatting a bit to: 
`__We will be joined by Jordan Henkel, first author of the paper!__` -> `This time we will be joined by *Jordan Henkel*, one of the authors of the paper!`